http/internal/logic/social/getFriendListLogic.go

```diff
@@ -34,6 +34,33 @@ func (l *GetFriendListLogic) GetFriendList(req *types.GetFriendListRequest) (res
 	res, err := l.svcCtx.ContactRpc.GetFriendsList(l.ctx, &contact.GetFriendsListRequest{
 		UserId: queryId,
 	})
+	var users []types.User
+
+	for _, friendID := range res.FriendsId {
+		userInfo, err := l.svcCtx.UserRpc.QueryById(l.ctx, &user.QueryByIdRequest{
+			UserId: friendID,
+		})
+		if err != nil {
```

Comment:
I consider ErrUserNotFound acceptable, but other error indicate that something goes wrong, please handle to types of error separately